torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2009,8 +2006,9 @@ at::Tensor AtenXlaType::mul(const at::Tensor& self, at::Scalar other) {
 
 at::Tensor& AtenXlaType::mul_(at::Tensor& self, const at::Tensor& other) {
   XLATensor self_tensor = bridge::GetXlaTensor(self);
-  XLATensor::mul_(self_tensor,
-                  bridge::GetOrCreateXlaTensor(other, self_tensor.GetDevice()));
+  XLATensor other_tensor =
```

Comment:
Why this change?

Reply:
Yea I forgot to revert this change. Nice catch!